### PR TITLE
Attach binaries to new releases

### LINKED ISSUE #17: Publish binaries on new release using CI
So there's a cool project called [trust](https://github.com/japaric/trust) that you can setup in your project, so that every time you create a new release/create a new tag, it will automatically build binaries for you using travis ci and publish them to the releases tab. This is really nice for people who want to use rust projects but don't want to have to deal with its compile times :D

edit: Looks like that project might be abandoned, but the idea is still the same. Ripgrep and Alacritty both do some sort of similar thing for their projects, so we could always check those out too.

### PR DESCRIPTION
**Pull request by kennycallado**
Hi,

Action to build the binaries. It's triggered on new releases or tags (only those with v before the version number) and attaches the binaries to them.

Fix #17.

### CODE REVIEW

File: .github/workflows/autometed-build.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
+name: Automated build
```
Comment (kamiyaa) [inline .github/workflows/autometed-build.yml:1]: File should be `automated-build` right?

Comment (kennycallado) [reply .github/workflows/autometed-build.yml:1]: Totally right 🙃

**Comment by kennycallado** [comment]
I should have given more details with PR, but I'm not very proud of my English 😅. 

The v prefix is not required if you release from github. You can create a new release with or without v and the action is going to start the build process to attach the binaries. 

There is another way to proceed. You can tag locally `git tag v0.9.3` (note the v) and push this tag to github `git push --tags`. Github stores the tag as expected, but the action is going to create a release (which is editable) and attach the binaries to it. 

However, there is the option to tag without creating a release. If the tag doesn't have v at the beginning, it is going to be stored but it won't fire the action. 

Then you can have tags like '0.9.3.1' which is not a release, it is just a tag, and others like 'v0.9.4' which triggers the action and creates a release. But you are also able to create a release like '0.9.3.1' (no v) if you release from github. Then to keep consistency, I think could be better to prefix the releases with v and tags without v.

Released from github:
screenshot-1

Released from tag:
screenshot-2

Just a tag:
screenshot-3

I hope I have explained it well.

**Comment by kamiyaa** [comment]
I see! Thanks for the explanation!

**Pull request merged**